woocommerce/woocommerce-gutenberg-products-block
# Add single product block template

For WooCommerce merchants to be able to fully utilize block themes, we need the most important WooCommerce core PHP templates to be available as block templates.

This PR adds a block template version of the [single-product.php template](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/single-product.php).

Fixes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5032

### Manual Testing

How to test the changes in this Pull Request:

**Prerequisite**: Please ensure you have a Block Template Theme activated such as TT1 and also the Gutenberg Plugin installed.

1. Load the Site Editor (Appearance > Editor) and select the `Single Product Page` template from the `General templates` section.
2. Confirm you can load the template. It will have a temporary placeholder block. This SVG will be replaced in a future PR.
3. Confirm the behavior on the frontend product page is identical to the behavior on the product page without this PR applied.

Once you have followed the above steps, add the same template files to your `theme-dir/block-templates` and change the file contents so you can differentiate between the Woo Block template and the Theme template. Repeat the above steps starting from Step 2 and confirm the Theme template is preferred over the Woo Blocks template

### Changelog

> FSE: Add single product block template.
